iai-callgrind/build.rs

```diff
@@ -39,9 +42,21 @@ mod imp {
         println!("cargo:rustc-cfg=client_requests_support=\"{value}\"");
     }
 
-    fn build_native() {
+    fn include_dirs(target: &Target) -> impl Iterator<Item = String> {
+        [
+            Cow::Owned(format!(
+                "IAI_CALLGRIND_{}_VALGRIND_INCLUDE",
+                target.triple.replace('-', "_")
```

Comment:
I'd prefer the environment variable in upper case.

Reply:
trying to edit this evening